packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -478,11 +482,86 @@ function useProcessMedia({
     ]
   );
 
+  /**
+   * Crop hidden video using FFmpeg.
+   *
+   * @param {import('@googleforcreators/media').Resource} resource Resource object.
+   */
+  const cropHiddenVideo = useCallback(
+    ({ id: elementId, resource: oldResource }, cropParams) => {
+      const { id: resourceId, src: url, mimeType } = oldResource;
+      const { newWidth, newHeight, cropElement } = cropParams;
+
+      const onUploadSuccess = ({ id, resource }) => {
+        copyResourceData({ oldResource, resource });
+        updateOldTranscodedObject(oldResource.id, resource.id, 'source-image');
+
+        updateElementById({
+          elementId,
+          properties: {
+            x: cropElement.x < 0 ? 0 : cropElement.x,
+            y:
+              cropElement.y < DANGER_ZONE_HEIGHT
+                ? -DANGER_ZONE_HEIGHT
+                : cropElement.y,
+            width: newWidth,
+            height: newHeight,
+          },
```

Comment:
I just cropped an image and the image jumped a bit on the y axis afterwards:


https://user-images.githubusercontent.com/841956/187660800-ab5d7bbe-c646-4363-a4c9-756172fc0893.mov